149358
Test: public void test36() throws Throwable { NNTP nNTP0 = new NNTP(); // Undeclared exception!  nNTP0.next();   }
Code: public int next() throws IOException { return sendCommand(NNTPCommand.NEXT); }
[[0.10059641 0.06380884 0.11875241 0.238976   0.0517805  0.13326083
 0.11060245 0.13955247 0.04267014]
[0.07074797 0.05387814 0.12422092 0.22147483 0.05240673 0.14348912
 0.11218704 0.18891004 0.03268524]
[0.06766701 0.03932568 0.15280539 0.23584747 0.05095971 0.15455249
 0.11434248 0.1504108  0.03408894]
[0.06600042 0.04194995 0.116207   0.33630726 0.03925209 0.14498548
 0.10524668 0.12080111 0.02924999]
[0.07939066 0.04506778 0.10125828 0.308684   0.04227638 0.13070756
 0.11221825 0.15175262 0.02864448]
[0.10410878 0.05148732 0.11701688 0.18762848 0.06721039 0.15106578
 0.12194779 0.14366585 0.05586874]
[0.07313619 0.0450798  0.11328106 0.28480577 0.05410964 0.13488582
 0.10757191 0.14975354 0.03737624]
[0.08480722 0.05588487 0.11864123 0.2730804  0.0545925  0.12400728
 0.10265691 0.14343502 0.04289462]
[0.06411332 0.04904281 0.12110242 0.2708673  0.04921025 0.15347616
 0.13676165 0.12185457 0.03357151]]